### ISSUE TITLE
Test notebooks with `NBInclude`? 

### ISSUE THREAD

**Issue by rdeits** [arsenal9971/LightFields.jl#1]
I enjoyed your talk at JuliaCon just now! 

I wanted to play around with the package, but I noticed that the notebooks are a bit out of date (the file paths refer to `../Diagrams/results/new_EPIs` instead of `../data_samples/results/EPIs`, for example). You might want to add a `runtests.jl` file that tests those notebooks. The easiest way to do that is with https://github.com/stevengj/NBInclude.jl . Your `test/runtests.jl` could look like: 

```julia
using NBInclude
@nbinclude("../notebooks/your_notebook.ipynb")
@nbinclude("../notebooks/your_other_notebook.ipynb")
```

**Comment by arsenal9971** [comment]
You are right!!, I need to write real examples and tests, I am planning to do it this week. I actually am resilient now to have notebooks on the repo since they are quite heavy. I will let  you know when this is done.
